lms/djangoapps/grades/tests/test_models.py

```diff
@@ -89,7 +89,10 @@ def test_creation(self):
         Happy path test to ensure basic create functionality works as expected.
         """
         vblocks = VisibleBlocks.objects.create_from_blockrecords([self.record_a])
-        expected_json = json.dumps([self.record_a._asdict()], separators=(',', ':'), sort_keys=True)
+        list_of_block_dicts = [self.record_a._asdict()]
+        for block_dict in list_of_block_dicts:
+            block_dict['locator'] = unicode(block_dict['locator'])  # BlockUsageLocator is not json-serializable
+        expected_json = json.dumps(list_of_block_dicts, separators=(',', ':'), sort_keys=True)
```

Comment:
If we end up using this pattern more, it would be good to customize the `._asdict()` method, but since it's just the once, this is fine.